tools/regression_test.py

```diff
@@ -0,0 +1,686 @@
+                    metric_value, fps = \
+                        test_backends(deploy_cfg=deploy_cfg,
+                                      model_cfg=model_cfg,
+                                      checkpoint_path=convert_checkpoint_path,
+                                      device=device_type,
+                                      metrics_name=metric_name,
+                                      logger=logger,
+                                      metric_info_dict=metric_info_dict)
+
+                    metric_name = metric_info_dict. \
+                        get(metric_name, {}).get('meta_name', None)
+                    if metric_name is None:
+                        logger.error(f'metrics_eval_list: {metrics_eval_list} '
+                                     'has not info name')
+                    assert metric_name is not None
+
+                    metric_list.append({metric_name: metric_value})
+                    metric_pytorch = pytorch_metric.get(str(metric_name))
+                    metric_tolerance_value = metric_tolerance.get(metric_name)
+                    if (metric_value - metric_tolerance_value) <= \
+                            metric_pytorch <= \
+                            (metric_value + metric_tolerance_value):
+                        test_pass = True
+                    else:
+                        test_pass = False
+            else:
+                for metric in metric_name_list:
+                    metric_list.append({metric: '-'})
+                test_pass = True if convert_result else False
+
+            # update useless metric
+            metric_useless = set(metric_all_list) - set(metric_name_list)
+            for metric in metric_useless:
+                metric_list.append({metric: '-'})
+
+            # update the report
+            update_report(
+                report_dict=report_dict,
+                model_name=model_cfg_path.parent.name,
+                model_config=str(model_cfg_path),
+                model_checkpoint_name=str(checkpoint_path),
+                dataset='',
+                backend_name=backend_name,
+                deploy_config=str(deploy_cfg_path),
+                static_or_dynamic=infer_type,
+                conversion_result=str(convert_result),
+                fps=fps,
+                metric_info=metric_list,
+                test_pass=str(test_pass)
+            )
+
+
+def save_report(report_info, report_save_path, logger):
+    """Convert model to onnx and then get metric.
+
+    Args:
+        report_info (dict):  Report info dict.
+        report_save_path (Path): Report save path.
+        logger (logging.Logger): Logger.
+    """
+    logger.info(f'Save regression test report '
+                f'to {report_save_path}, pls wait...')
+
+    df = pd.DataFrame(report_info)
+    df.to_excel(report_save_path)
+
+    logger.info(f'Saved regression test report '
+                f'to {report_save_path}.')
+
+
+def main():
+    args = parse_args()
```

Comment:
does `args.test_type` work in this version? what's the result table look like if only test conversion?

Reply:
Done. It will show `-` in the `fps` and `metric`, but will show `True` when conver is success